pom.xml

```diff
@@ -42,8 +42,8 @@
     <version.springfox>2.9.2</version.springfox>
     <version.metis>9-SNAPSHOT</version.metis>
     <version.europeana>2.15.3</version.europeana>
-    <version.postgresql>42.3.3</version.postgresql>
-    <version.aws.s3>1.11.762</version.aws.s3>
+    <version.postgresql>42.3.7</version.postgresql>
```

Comment:
Why not 42.5.0 as the ticket describes?

Reply:
used intelliJ autocomplete, by mistake. Updated to the right version.